testsuite/generate-format/src/compute.rs

```diff
@@ -0,0 +1,40 @@
+// Copyright (c) The Libra Core Contributors
+// SPDX-License-Identifier: Apache-2.0
+
+use generate_format::{add_deserialization_tracing, add_proptest_serialization_tracing, FILE_PATH};
+use serde_reflection::Tracer;
+use serde_yaml;
+use std::{fs::File, io::Write};
+use structopt::StructOpt;
+
+#[derive(Debug, StructOpt)]
+#[structopt(
+    name = "Libra format generator",
+    about = "Trace serde (de)serialization to generate format descriptions for Libra types"
+)]
+struct Options {
+    #[structopt(short, long)]
+    with_deserialize: bool,
+
+    #[structopt(short, long)]
+    record: bool,
+}
+
+fn main() {
+    let options = Options::from_args();
+
+    let mut tracer = Tracer::new(lcs::is_human_readable());
+    tracer = add_proptest_serialization_tracing(tracer);
```

Comment:
I remember reading in the rust book that it is preferable to not import functions but modules or structs.

Reply:
Point taken. I will likely extend/refactor this part in the next diffs